event-driven-architecture/src/main/java/com/iluwatar/eda/App.java

```diff
@@ -0,0 +1,41 @@
+package com.iluwatar.eda;
+
+import com.iluwatar.eda.event.Event;
+import com.iluwatar.eda.event.UserCreatedEvent;
+import com.iluwatar.eda.event.UserUpdatedEvent;
+import com.iluwatar.eda.handler.UserCreatedEventHandler;
+import com.iluwatar.eda.handler.UserUpdatedEventHandler;
+import com.iluwatar.eda.model.User;
+
+/**
+ * An event-driven architecture (EDA) is a framework that orchestrates behavior around the
```

Comment:
Yes from what i've been reading the reactor pattern is based on EDA and is also closely related to non-blocking operations. 
http://stackoverflow.com/questions/9138294/what-is-the-difference-between-event-driven-model-and-reactor-pattern.

My toughts regarding MDA vs EDA ... in an Event-driven Architecture `Events` (occurences within a system) are generated and passed around. A Message Driven System can be used to pass around these `Events`, although other types of messages can be passed around too.